lib/active_remote/serializers/protobuf.rb

```diff
@@ -1,24 +1,26 @@
+require "active_remote/typecasting"
+
 module ActiveRemote
   module Serializers
     module Protobuf
       extend ActiveSupport::Concern
 
       TYPECASTER_MAP = {
```

Comment:
You're calling this constant `TYPECASTER_MAP`, but the one in typecasting.rb seems to be more of the `TYPECASTING_MAP`. This seems to deal more in setting types specifically on proto fields.